src/Wellcome.Dds/Wellcome.Dds.Dashboard/appsettings.Staging.json

```diff
@@ -45,28 +48,28 @@
     "DashBodyInject": "background-image:url(/img/staging.png); background-repeat;",
     "GoobiCall": "DO NOT CALL"
   },
-    "Storage-Production": {
-        "StorageApiTemplate": "https://api.wellcomecollection.org/storage/v1/bags/digitised/{0}",
-        "TokenEndPoint": "https://auth.wellcomecollection.org/oauth2/token",
-        "Scope": "https://api.wellcomecollection.org/storage/v1/bags",
-        "StorageApiTemplateIngest": "https://api.wellcomecollection.org/storage/v1/ingests",
-        "ScopeIngest": "https://api.wellcomecollection.org/storage/v1/ingests",
-        "PreferCachedStorageMap": false,
-        "MaxAgeStorageMap": 180
-    },
-    "BinaryObjectCache": {
-      "StorageMaps": {
-        "AvoidCaching": false,
-        "AvoidSaving": false,
-        "WriteFailThrowsException": true,
-        "Container": "wellcomecollection-stage-iiif-storagemaps",
-        "Prefix": "stgmap-",
-        "MemoryCacheSeconds": 180
-      }
-    },
-    "AzureAd": {
-        "Instance": "https://login.microsoftonline.com/",
-        "Domain": "wellcomecloud.onmicrosoft.com",
-        "CallbackPath": "/signin-oidc"
+  "Storage": {
+    "StorageApiTemplate": "https://api-stage.wellcomecollection.org/storage/v1/bags/digitised/{0}",
+    "TokenEndPoint": "https://auth.wellcomecollection.org/oauth2/token",
+    "Scope": "https://api.wellcomecollection.org/storage/v1/bags",
+    "StorageApiTemplateIngest": "https://api-stage.wellcomecollection.org/storage/v1/ingests",
+    "ScopeIngest": "https://api.wellcomecollection.org/storage/v1/ingests",
+    "PreferCachedStorageMap": false,
+    "MaxAgeStorageMap": 180
+  },
+  "BinaryObjectCache": {
+    "StorageMaps": {
+      "AvoidCaching": false,
+      "AvoidSaving": false,
+      "WriteFailThrowsException": true,
+      "Container": "wellcomecollection-stage-iiif-storagemaps",
+      "Prefix": "stgmap-",

```

Comment:
should this differ for stg + stgprd?

Reply:
No, the bucket is enough - the prefix is used by an IMemoryCache for short term caching, which could in theory lead to collisions between same keys in different buckets.